docs/conf.py

```diff
@@ -174,14 +166,13 @@
 napoleon_google_docstring = False
 
 # -- Options for the edit_on_github extension ----------------------------------------
-extensions.remove('astropy_helpers.extern.numpydoc')
```

Comment:
we still need to remove this, jut from `astropy_sphinx` instead. It's possible this is what's causing the doc warning.

Reply:
But extension in `astropy_sphinx` does not contain `astropy_helpers.extern.numpydoc`, will raise an error if removed . Also `numpydocs` should be removed right ?

Reply:
@Cadair ping

Reply:
yeah in `astropy_sphinx` it looks like it's just `numpydoc` we need to pop (https://github.com/astropy/sphinx-astropy/blob/master/sphinx_astropy/conf/v1.py#L135)